app/views/chargeback/_cb_rate_edit_table.html.haml

```diff
@@ -84,10 +101,30 @@
             = text_field_tag("variable_rate_#{detail_index}_#{tier_index}", tier[:variable_rate],
               :maxlength => MAX_NAME_LEN, "data-miq_observe" => {:interval => '.5', :url => url}.to_json)
           %td.action
-            = button_tag(_("Delete"), :class => "btn btn-default", :alt => t = _("Remove the tier"), :title => t,
-                                      :onclick => "miqAjaxButton('#{url_for(:action => "cb_tier_remove",
-                                                                            :index => "#{detail_index}-#{tier_index}",
-                                                                            :button => "remove")}');")
+            .dropdown.dropdown-kebab-pf
+              %button.btn.btn-link.dropdown-toggle{:type => "button",
+                                                   :id => "dropdownKebab",
```

Comment:
@amaurygonzalez we need to get rid off id called `dropdownKebab` and and use any unique value of id attribute,
something like this  "dropdown_kebab_#{detail_index}_#{tier_index}" (if usable) because it must be unique as is ibn post and mainly QE guys are not able to write test for such elements

thanks